docs/processes/release/tasks/integration.md

```diff
@@ -143,11 +143,14 @@ should be investigated and fixed.
 To do this, build and test normally, then replace the ICU data shared library
 with the stubdata library and run the tests again with the -w option.
 
-On Linux (adjust the version number, 60.1 in this example, as required)
+Using an in-source build on Linux:
 
 ```sh
 cd icu4c/source
-cp stubdata/libicudata.so.60.1 lib/
+./runConfigureICU Linux
+make -j2 check
+rm lib/libicudata.so*
+cp -P stubdata/libicudata.so* lib/
 cd test/intltest
 INTLTEST_OPTS=-w make check
 cd ../cintltst
```

Comment:
*after* this line please add something like the following (finishes what the ticket requests)
```

For debugging (for example using gdb) you cannot `make check`. You need to set the `LD_LIBRARY_PATH` for the lib folder, and then run `./cintltst -w` or `./intltest -w` etc. in the debugger.

Example for one possible folder layout of an out-of-source build: `export LD_LIBRARY_PATH=../../lib:../../stubdata:../../tools/ctestfw`
```

Reply:
Done.